documentation/modules/con-overview-running-kafka-bridge.adoc

```diff
@@ -19,4 +19,10 @@ Configure and deploy the Kafka Bridge as a `KafkaBridge` resource.
 You'll need a running Kafka cluster that was deployed by the Cluster Operator in a Kubernetes namespace.
 You can configure your deployment to access the Kafka Bridge outside the Kubernetes cluster.
 
+It's important to note that each instance of the Kafka Bridge maintains its own set of in-memory consumers (and subscriptions) that connect to the Kafka Brokers on behalf of the HTTP clients.
+This means that each HTTP client must maintain affinity to the same Bridge pod in order to access any subscriptions that are created. 
+Therefore, **it is not recommended to run more than 1 replica of the Kafka Bridge per Kubernetes Deployment**. 
+Additionally, when an instance of the Kafka Bridge restarts (for example, if Kubernetes moves the workload to another node), the in-memory consumers and subscriptions are lost.
+**It is the responsibility of the HTTP client to recreate any consumers/subscriptions if the Bridge restarts.** 
```

Comment:
Apart the third sentence `Therefore, **it is not recommended to run more than 1 replica of the Kafka Bridge per Kubernetes Deployment**. ` ... all the others are not strictly related to Kubernetes. I mean, they are valid even in the context of running the Kafka bridge on bare metal/virtual machines, where we can talk about bridge "instance" instead of bridge "pod".
Tbh I would move them in the main chapter "[Running the Kafka Bridge](https://strimzi.io/docs/bridge/latest/#overview-components-running-kafka-bridge-bridge)" and leaving just a not about the third sentece here. Wdyt?

Reply:
I think that makes perfect sense! I'll work on moving it over.